spec/metrics/relation_spec.rb

```diff
@@ -237,6 +257,26 @@
         end
       end
     end
+
+    context "complex queries" do
+      it "group + where" do
+        expect(rel.count('user_id').group(:traffic_source).fill(0).where(user_id: 123).past('28d').to_sql)
+          .to eq "select count(user_id) from \"dummy\" where (user_id=123) and (time > now() - 28d) " \
```

Comment:
Style/DotPosition: Place the . on the previous line, together with the method call receiver.